Applications/DataExplorer/DataView/DiagramView/QArrow.h

```diff
@@ -25,26 +25,26 @@ const double PI = 3.14159265;
 class QArrow : public QGraphicsItem
 {
 public:
-    QArrow(float l, float a, float hl, float hw, QPen& pen,
+    QArrow(qreal l, qreal a, qreal hl, qreal hw, QPen& pen,
            QGraphicsItem* parent = nullptr);
-    QArrow(float l, float a, QPen& pen, QGraphicsItem* parent = nullptr);
+    QArrow(qreal l, qreal a, QPen& pen, QGraphicsItem* parent = nullptr);
     ~QArrow() override;
 
     double getLength();
     double getAngle();
     void paint(QPainter* painter, const QStyleOptionGraphicsItem* option,
                QWidget* widget) override;
     QRectF boundingRect() const override;
-    void setAngle(double a);
-    void setLength(double l);
+    void setAngle(float a);
```

Comment:
Why isn't the argument type qreal?

Reply:
fixed (not sure why I didn't get a warning here)